src/mode_manager.ts

```diff
@@ -19,7 +19,8 @@ export class ModeManager implements Disposable, NeovimRedrawProcessable, NeovimE
      * True when macro recording in insert mode
      */
     private isRecording = false;
-
+    public neovimToggle = true;
```

Comment:
Can we make this more descriptive. How about?

```suggestion
    public neovimToggleStayInsert = true;
```

Reply:
"stay in insert" is not really a correct description, it's more of a side-effect. The actual thing being done is that vscode-neovim is being disabled. Right?

Reply:
both in a way right. To actually disable the extension, one must go into insertmode after switched the toggle. So currently it describes the name @trkoch want to give it.
So the Toggle would need also to send a "GoToInsertMode"-Request.
For understandability and a clean feature @justinmk 's approach would be better. 
If you both agree i go with neovimToggle(since neovimEnable is not really what it does, it also disables the Extension. And neovimToggleStayInsert is what it does but doesn't reflect the intent)

Reply:
If the intention is to toggle (almost) all vscode-neovim functionality, then let's name it as such. Whatever internal compromises are made (hopefully temporary) are irrelevant.

Reply:
Thank you for your insight! Upon reflection I agree.

> The actual thing being done is that vscode-neovim is being disabled.